cron/poll_read_issue_comments.py

```diff
@@ -301,13 +311,17 @@ def poll_read_issue_comments(api):
     # If page=all, you have to loop through pages as well
     for page in paged_results:
         for issue_comment in page:
-            # Get info and store in db
-            issue_id = issue_comment["issue_id"]
-            global_comment_id = str(issue_comment["global_comment_id"])
-            comment_text = issue_comment["comment_text"]
-            if is_command(comment_text):
-                insert_or_update(api, global_comment_id, issue_id, comment_text)
 
+            # General exception catching
+            try:
+                # Get info and store in db
+                issue_id = issue_comment["issue_id"]
+                global_comment_id = str(issue_comment["global_comment_id"])
+                comment_text = issue_comment["comment_text"]
+                if is_command(comment_text):
+                    insert_or_update(api, global_comment_id, issue_id, comment_text)
+            except Exception as e:
+                __log.exception("Some exception occurred")
```

Comment:
`log.exception` already prints the stack trace. I have no idea what exception would occur there, but I don't want to fatally crash the code